streams/src/test/java/org/apache/kafka/streams/integration/SuppressionDurabilityIntegrationTest.java

```diff
@@ -103,7 +103,7 @@ public static void closeCluster() {
     private static final long COMMIT_INTERVAL = 100L;
 
     @ParameterizedTest
-    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
+    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE})
     public void shouldRecoverBufferAfterShutdown(final String processingGuarantee, final TestInfo testInfo) {
```

Comment:
As above: let's remove `processingGuarantee` entirely, given that we have only one parameter value.